src/ripple/nodestore/impl/Shard.h

```diff
@@ -128,11 +129,14 @@ class Shard final
         verified backend data.
         @param referenceHash If present, this hash must match the hash
         of the last ledger in the shard.
+        @param writeDeterministicShard If true, shard will be rewritten
+        deterministically.
     */
     bool
     finalize(
         bool const writeSQLite,
-        boost::optional<uint256> const& referenceHash);
+        boost::optional<uint256> const& referenceHash,
+        const bool writeDeterministicShard = true);
```

Comment:
We usually leave `const` off of value params

Reply:
Done.

Reply:
It looks like the `const` is still there.

Reply:
Done.